src/Tickets/Admin/Glance_Items.php

```diff
@@ -13,26 +13,40 @@
  */
 class Glance_Items {
 
+	/**
+	 * The key for the transient that stores the attendee count.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	private string $attendee_count_key = 'tec_tickets_glance_item_attendees_count';
+
 	/**
 	 * Method to register glance items related hooks.
 	 *
 	 * @since 5.5.10
 	 */
 	public function hooks() {
 		add_filter( 'dashboard_glance_items', [ $this, 'custom_glance_items_attendees' ], 10, 1 );
+		add_action( 'tec_tickets_update_glance_item_attendee_counts', [ $this, 'update_attendee_count' ] );
 	}
 
 	/**
 	 * Custom glance item for Attendees count.
 	 *
+	 * @since TBD Make use of transients and cron jobs to avoid performance issues.
+	 *
 	 * @param array $items The array of items to be displayed.
 	 * @return array $items The maybe modified array of items to be displayed.
 	 */
 	public function custom_glance_items_attendees( $items = [] ): array {
-		$results = Tribe__Tickets__Tickets::get_attendees_by_args( [] );
-		$total   = count( $results['attendees'] );
+		$total = get_transient( $this->attendee_count_key );
```

Comment:
Call `update_attendee_count()` directly here too and get that value from it to use.

Reply:
Sounds good.